app/code/core/Mage/Core/Model/Config.php

```diff
@@ -422,6 +423,23 @@ public function loadDb()
         return $this;
     }
 
+    /**
+     * Load environment variables and override config
+     *
+     * @return self
+     */
+    public function loadEnv(): Mage_Core_Model_Config
+    {
+        if ($this->_isLocalConfigLoaded && Mage::isInstalled()) {
+            Varien_Profiler::start('config/load-env');
+            /** @var Mage_Core_Helper_EnvironmentConfigLoader $environmentConfigLoaderHelper */
+            $environmentConfigLoaderHelper = Mage::helper('core/environmentConfigLoader');
+            $environmentConfigLoaderHelper->overrideEnvironment($this);
+            Varien_Profiler::stop('config/load-env');
```

Comment:
```suggestion
            /** @var Mage_Core_Helper_EnvironmentConfigLoader $environmentConfigLoaderHelper */
            $environmentConfigLoaderHelper = Mage::helper('core/environmentConfigLoader');
            $environmentConfigLoaderHelper->overrideEnvironment($this);
```

If I'm not mistaken we were kinda talking about removing the profiler calls since there are great tools now for profiling, just in case